### PR TITLE
Do not reload NAD when navigating between tabs

### PR DESCRIPTION
**Pull request by YenguiSeddik**

### CODE REVIEW

File: src/components/diagrams/networkAreaDiagram/network-area-diagram-pane.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ export function NetworkAreaDiagramPane({
     disabled,
     visible,
```
Comment (FranckLecuyer) [inline src/components/diagrams/networkAreaDiagram/network-area-diagram-pane.js:22]: property 'visible' no more used : to be removed

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]
Tests: OK

**Review by bendaamerahm — APPROVED** [review]
Test: Ok

File: src/components/diagrams/networkAreaDiagram/network-area-diagram-pane.js (hunk 1 above)
Comment (YenguiSeddik) [reply src/components/diagrams/networkAreaDiagram/network-area-diagram-pane.js]: ok, done.

File: src/components/diagrams/networkAreaDiagram/network-area-diagram-pane.js [hunk 2]
```diff
@@ -19,9 +19,8 @@ export function NetworkAreaDiagramPane({
     onClose,
     align,
     disabled,
-    visible,
```
Comment (FranckLecuyer) [inline src/components/diagrams/networkAreaDiagram/network-area-diagram-pane.js:22]: Remove 'visible' property also in PropTypes at the end of the file ...

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]

**Review by bendaamerahm — APPROVED** [review]

**Review by FranckLecuyer — APPROVED** [review]

**Pull request merged**